3444/2/.gitignore

```diff
@@ -0,0 +1,2 @@
+.env
```

Comment:
how it's possible to have it in git repo and in gitignore? :)

Reply:
it's possible cause i did commit with .env, then founded it in index and decided to add it into gitignore, after that made rebase)

Reply:
It doesn't make sense to have them both.

Reply:
read my comments about having .env.sample

Reply:
done